task/argocd-task-connect-repo/0.1/argocd-task-connect-repo.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: argocd-task-connect-repo
+  namespace: argocd
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Deployment
+    tekton.dev/tags: deploy
+    tekton.dev/displayName: "argocd"
```

Comment:
not important but should we change this displayName to something else and meaningful?

Reply:
Yes, sorry, good point. perhaps argocd-repository?